components/form-vue/control/content/content.ts

```diff
@@ -27,6 +27,9 @@ export default class AppFormControlContent extends BaseFormControlTS {
 	@Prop(Number)
 	minHeight!: number;
 
+	@Prop(String)
```

Comment:
startupArg is too vague. I can't figure out what it does. Should we call these "Do on startup" or something? "execute-on-startup"? not sure